qiita_pet/templates/sitebase.html

```diff
@@ -510,6 +511,23 @@
                 <a href="https://github.com/biocore/qiita/blob/master/README.rst#current-features">Current and Future Features</a>
               </li>
             </ul>
+            <ul class="nav navbar-nav">
+              <!-- downloads -->
+              <li class="dropdown">
+                <a href="#" data-toggle="dropdown" class="dropdown-toggle">Downloads<b class="caret"></b></a>
+                <ul class="dropdown-menu">
+                  <li>
+                    <a href="{% raw qiita_config.portal_dir %}/release/download/public">
+                      Public processed data
```

Comment:
Maybe instead "Public BIOM and metadata files", or something a bit more specific?